### PR TITLE
Fixed a minor typo

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

**Comment by cordoval** [comment]
:+1: :older_man:

File: components/filesystem/introduction.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ The Filesystem Component
 
 .. tip::
 
-    A lock handler feature was introduce in symfony 2.6.
+    A lock handler feature was introduced in symfony 2.6.
```
Comment (stof) [inline components/filesystem/introduction.rst]: shouldn't this say ``the feature`` rather than ``A feature`` ?

Comment (WouterJ) [inline components/filesystem/introduction.rst]: I think so

**Comment by javiereguiluz** [comment]
I've included the suggested change. This PR can now be considered *finished* and ready to be merged. Thanks.

**Comment by xabbuh** [comment]
:+1:

**Comment by timglabisch** [comment]
:+1:

**Comment by weaverryan** [comment]
Clearly better. Thanks Javier!

**Pull request merged**